config/zfsnapphpdebugbar.config.php

```diff
@@ -3,7 +3,7 @@
 return array(
     'php-debug-bar' => array(
         'view' => array(
-            'custom-style-path' => 'zf-snap-php-debug-bar.css',
+            'custom-style-path' => 'zf-snap-php-debug-bar.min.css',
```

Comment:
I don't get it. It will increase complexity of releasing and don't give any value for developers. It's very well practice, but for production, not for development.